app/bundles/LeadBundle/Translations/en_US/messages.ini

```diff
@@ -5,6 +5,7 @@ mautic.campaign.lead.field_value="Contact field value"
 mautic.campaign.lead.device="Contact device"
 mautic.campaign.lead.owner="Contact owner"
 mautic.campaign.lead.updatelead="Update contact"
+mautic.campaign.lead.updatecompany="Update contacts primary company"
```

Comment:
I think there should be an apostrophe in it like `Update contact's primary company`

(But maybe a confirmation from an English man would be nice since both of us use English as a second language)